src/Workspaces/Core/Portable/Classification/ClassificationTypeNames.cs

```diff
@@ -27,6 +27,7 @@ public static class ClassificationTypeNames
         public const string OperatorOverloaded = "operator - overloaded";
         public const string PreprocessorKeyword = "preprocessor keyword";
         public const string StringLiteral = "string";
+        public const string UTF8StringLiteral = "UTF8 string";
```

Comment:
i would remove thse and just use the existing classification names.  e.g. we don't distinguish integer-classification versus double-classification.  They're just 'numbers'.   So here i think it's fine, absent some really strong user feedback, that we reuse the existing classifications.